torchx/lib/torchx/backend.ex

```diff
@@ -143,6 +172,20 @@ defmodule Torchx.Backend do
     NIF.squeeze(to_ref(t)) |> from_ref(out)
   end
 
+  def expand(%T{} = t, shape) do
```

Comment:
Do we use `expand` anywhere?

Reply:
I introduced it while trying to implement broadcasting along axis, which does not have direct analogue in PyTorch.